src/services/item/plugins/published/repositories/itemPublished.ts

```diff
@@ -129,9 +131,10 @@ export const ItemPublishedRepository = AppDataSource.getRepository(ItemPublished
   // bug: does not take into account child items
   async getLikedItems(limit: number = 10): Promise<Item[]> {
     const itemPublished = await this.createQueryBuilder('item_published')
-      .leftJoinAndSelect('item_published.item', 'i')
-      .innerJoin('item_like', 'il', 'il.item_id = i.id')
-      .groupBy(['i.id', 'item_published.id'])
+      .innerJoinAndSelect('item_published.item', 'item')
+      .innerJoinAndSelect('item.creator', 'member')
+      .innerJoin('item_like', 'il', 'il.item_id = item.id')
+      .groupBy(['item.id', 'member.id', 'item_published.id'])
```

Comment:
Will this not separate the like count in multiple like count per user?
Like one count per unique value of (item.id, member.id). I guess it's okay because of having only one member per item (creator) in the query

Reply:
Yes I guess you are right... I was not sure about how to handle the `groupBy`. I was getting an error about a non-aggregated column when omitting it.